src/App.scss

```diff
@@ -1 +1,4 @@
 @import './src/Utilities.scss';
+#root {
```

Comment:
You won't need to set this if you use the appropriate `PageHeader` and `Main` Components